libs/remix-core-plugin/src/lib/compiler-fetch-and-compile.ts

```diff
@@ -134,7 +134,18 @@ export class FetchAndCompile extends Plugin {
     if (!data) {
       setTimeout(_ => this.emit('notFound', contractAddress), 0)
       this.unresolvedAddresses.push(contractAddress)
-      return localCompilation()
+      const compilation = await localCompilation()
+      if (compilation) {
+        let found = false
+        compilation.visitContracts((contract) => {
+          found = util.compareByteCode(codeAtAddress, '0x' + contract.object.evm.deployedBytecode.object)
+          return found
+        })
+        if (found) {
```

Comment:
This will always be `false` if the execution of `compilation.visitContracts` takes too long.

Reply:
the execution is sync here.